mlflow/R/mlflow/tests/testthat/test-tracking-experiments.R

```diff
@@ -66,6 +66,26 @@ test_that("mlflow_list_experiments() works properly", {
   # Returns NULL when no experiments found
   expect_null(mlflow_list_experiments("DELETED_ONLY"))
 
+  # experiment tags are returned if at least one experiment has tags
+  mlflow_set_experiment_tag("key2", "value2", experiment_id = ex2)
+  experiments <- mlflow_list_experiments()
+  expect_true("tags" %in% names(experiments))
+  expect_setequal(
+    experiments$tags, list(NULL, NULL, c(key = "key2", value = "value2"))
+  )
+
+  # experiment tags are returned if every experiment has tags
+  mlflow_set_experiment_tag("key1", "value1", experiment_id = ex1)
+  mlflow_set_experiment_tag("key0", "value0", experiment_id = "0")
```

Comment:
Can we add multiple tags to one of the experiments to verify that we can have a list of tags per experiment?